.github/workflows/installer-win.yml

```diff
@@ -2,6 +2,11 @@
 on:
   pull_request:
     paths:
+      - 'requirements/*.txt'
```

Comment:
```suggestion
      - 'requirements/*.yml'
```

Reply:
Perhaps you should also change this in all other places that reference `'requirements/*.yml'`?

Reply:
Should I also add the changes to the MacOS installer workflow @ccordoba12 ?

Reply:
Yes, please go ahead and do that too.